libfwupdplugin/rustgen.py

```
@@ -7,15 +7,22 @@
 import uuid
 import argparse
 
 from enum import Enum
+from pathlib import Path
 from typing import Optional, List, Tuple, Dict
 
 from jinja2 import Environment, FileSystemLoader, select_autoescape
 
 
+def file_next_to_module(file: str) -> str:
+    dir = Path(__file__).parent.relative_to(Path.cwd(), walk_up=True)
```

Ruff (0.15.15)

[error] Variable `dir` is shadowing a Python builtin

(A001)